### ISSUE TITLE
[DOCUMENTATION] TRAPP Update single

### ISSUE THREAD

**Issue by phh** [BenjaminMedia/bonnier-php-sdk#47]
https://github.com/BenjaminMedia/bonnier-php-sdk/blob/master/trapp.md#update-single

```
$translation = new \Bonnier\Trapp\ServiceTranslation($username, $secret);
$translation = $service->getById('55a8cb09214f48032700421f');
```

I guess this should be

```
$service = new \Bonnier\Trapp\ServiceTranslation($username, $secret);
$translation = $service->getById('55a8cb09214f48032700421f');
```

**Comment by skipperbent** [comment]
Thanks for the feedback :+1: 

This issue has been resolved and will be available in the next release. 

Added in pull request: #48

**Comment by skipperbent** [comment]
Fixed in release 2.7.1
